app/frontend/src/components/Navigation/Navigation.tsx

```diff
@@ -115,13 +117,22 @@ export default function Navigation() {
   const authClasses = useAuthStyles();
   const authenticated = useAuthContext().authState.authenticated;
   const [open, setOpen] = React.useState(false);
+  const { data } = useNotifications();
+    
+  let notificationNumber = data?.unseenMessageCount;
```

Comment:
Make this `const` and also make it the sum of the unread messages and also host requests, since they are under this page too.